openc3-cosmos-cmd-tlm-api/app/controllers/script_autocomplete_controller.rb

```diff
@@ -215,9 +215,13 @@ def build_cmd_snippet(packet, target_info)
       params = filtered_items.each_with_index.map do |item, index|
         if item['data_type'] == 'BLOCK'
           # Unpack binary data to uppercase hex string
-          default = "0x#{item['default'].unpack('H*').first.upcase}"
+          if item['default'] and String === item['default']
+            default = "0x#{item['default'].unpack('H*').first.upcase}"
+          else
+            default = "''"
+          end
         elsif item['data_type'] == 'STRING'
-          default = "'#{item['default']}'" || ''
+          default = "'#{item['default']}'"
```

Comment:
Was this to address a specific issue with BLOCK vs STRING?

Reply:
That or never did anything so I removed it.